Gems/Atom/RHI/Vulkan/Code/Include/Atom/RHI.Loader/FunctionLoader.h

```diff
@@ -29,16 +29,12 @@ namespace AZ
             bool Init();
             void Shutdown();
 
-            virtual ~FunctionLoader();
```

Comment:
Was this removed on purpose?

Reply:
As I mentioned in the comment before, `FunctionLoader` is not an interface anymore, so there is no need for the (negative performance impact of a) virtual destructor.